apps/demos/Demos/Chat/Customization/Angular/app/app.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="demo-container">
+  <div class="chat-container">
+    <dx-chat
+      #chat
+      [user]="currentUser"
+      [items]="messages$ | async"
+      [showAvatar] = "true"
+      [showUserName] = "true"
+      [showDayHeaders] = "true"
+      [showMessageTimestamp] = "true"
```

Comment:
so, the value if this chat's property is always true and never changed ..
It should be bound to the state

Reply:
[a40d776](https://github.com/DevExpress/DevExtreme/pull/28482/commits/a40d77698d64bcb42c0340010e88cdccd0628c50)